ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -102,6 +102,49 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlPostWithSession(SteamCommunityURL, request, data, referer).ConfigureAwait(false);
 		}
 
+		internal async Task AcceptDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
+			if (response == null) {
+				Bot.ArchiLogger.LogNullError(nameof(response));
```

Comment:
This is unacceptable, you're going to throw null error (aka what we need to correct) on the most common request timeout. I don't know why you didn't pick up convention from literally everywhere else, but this needs to be corrected.

Reply:
Strings.WarningFailed will be enough, right?

Reply:
Once you split this method into two parts (AWH and Bot, like I explained below) you should just return `null` in place of `HashSet<ulong>`. Failed requests already carry the appropriate warning returned by `WebBrowser`, so there is no point in duplicating it. Check another functions of AWH for an idea how to handle it - `GetInventory()` for example.